### DISCUSSION TITLE
Anyone else having problems with Google account sign in in private window?

### DISCUSSION THREAD

**Discussion by m-chrzan** [qutebrowser/qutebrowser#5862] [Q&A] [answered]
When trying to log in to a Google account from a private qutebrowser window, before being able to enter a password, Google informs me that I can't log in from my browser:

>Couldn't sign you in
>This browser or app may not be secure. [Learn more](https://support.google.com/accounts/answer/7675428?hl=en)
>Try using a different browser. If you’re already using a supported browser, you can refresh your screen and try again to sign in.

Logging in in a non-private qutebrowser window works (or at least via the "Add another account" flow, having logged in to an account before I started noticing this behavior in private).

JavaScript is running, I don't have any plug-ins. Is this a known issue, can anyone reproduce this problem? If this is a real issue, is it known what qutebrowser would have to do to comply with whatever checks Google is using to block "insecure" (by their estimation) browsers?

For context, I'm using

    qutebrowser v1.14.0 
    Backend: QtWebEngine (Chromium 80.0.3987.163)
    Qt: 5.15.1

on Arch Linux.

**Comment by The-Compiler** [comment, marked as answer]
Can you try if it works on the very first private window you open after starting qutebrowser, but then breaks when you close that private window and open a fresh one? I suspect this is the same as #5810 which causes our workaround for #5182 to break.

**Reply by m-chrzan** [reply]
Ah yes, exactly the behavior you describe.
